src/m365/spo/commands/page/page-control-get.ts

```diff
@@ -52,14 +52,16 @@ class SpoPageControlGetCommand extends SpoCommand {
 
         if (control) {
           const controlData = {
+            id: control.id,
+            type: getControlTypeDisplayName(
+              control.controlType || 0
+            ),
+            title: control.webPartData?.title,
             controlType: control.controlType,
             order: control.position.sectionIndex,
-            id: control.id,
             controlData: {
               ...control
-            },
-            type: getControlTypeDisplayName(control.controlType || 0),
-            ...control.webPartData || {}
```

Comment:
Rather than removing the whole webPartData object from the result, could we remove just the ID to avoid it overwriting the one on the control?

Reply:
Sure! 
The reason I removed it alltogether was because the resulting object already has the `webPartData` property as part of the `controlData` property. So, in the previous version all webPartData properties were **copied**/duplicated to the root object and the original object is also still part of the **controlData**... 
I focussed my change to put the three default properties `['id', 'type', 'title']` part of the resulting JSON of the `list` command and leave the rest of the properties the same as they're returned by the API.
Would you rather have it as it was before?

Reply:
Understood. Thank you for clarifying. Since we're moving to a major version, we can justify the breaking change.